Readability.js

```diff
@@ -102,7 +102,7 @@ Readability.prototype = {
   DEFAULT_MAX_PAGES: 5,
 
   // Element tags to score by default.
-  DEFAULT_TAGS_TO_SCORE: ["SECTION", "P", "TD", "PRE"],
+  DEFAULT_TAGS_TO_SCORE: "section,h2,h3,h4,h5,h6,p,td,pre".toUpperCase().split(","),
```

Comment:
The idea is to score article intermediary headings in orther to better identify which part of page contents actually define the main article. We were only scoring P, TD and PRE tags; I think intermediary headings are definitely important parts of an article, carrying meaning and semantical weight. This patch simply makes the parser scoring them, and this fix the issue with the test page because the main content area is now properly detected, thanks to scoring these headings by default.